wpimath/src/main/java/edu/wpi/first/math/geometry/Rotation2d.java

```diff
@@ -245,6 +249,15 @@ public double getTan() {
     return m_sin / m_cos;
   }
 
+  /**
+   * Returns the rotation matrix representation of the Rotation2d.
+   *
+   * @return A rotation matrix.
+   */
+  public Matrix<N2, N2> getRotationMatrix() {
```

Comment:
I don't see a use case for this. It's best to keep things in terms of Rotation2d. We only provide an euler angle getter because it's extremely widely used (tho Euler angles suck, mathematically).